### ISSUE TITLE
.NET 4.7.2 build does not work

### ISSUE THREAD

**Issue by odalet-addup** [addupsolutions/addup-xslt#1]
Here is what we get when running  **xslt.exe** (from the .NET 4.7.2 build):
```
Exception non gérée : System.TypeInitializationException: Une exception a été levée par l'initialiseur de type pour '<Module>'. ---> System.IO.FileNotFoundException: Impossible de charger le fichier ou l'assembly 'System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' ou une de ses dépendances. Le fichier spécifié est introuvable.
   à .cctor()
   --- Fin de la trace de la pile d'exception interne ---
```

**Comment by odalet-addup** [comment]
Oops, must be because we build with **dotnet.exe**...

**Comment by odalet-addup** [comment]
Fixed by 5a02f34584babd2cdc3f4b11a4ff9974c1851512
